CHANGELOG.rst

```diff
@@ -26,6 +31,7 @@ Internal changes
 ^^^^^^^^^^^^^^^^
 * Changed the French translation of "wet days" from "jours mouillés" to "jours pluvieux". (:issue:`1825`, :pull:`1826`).
 * In order to adapt to changes in `pytest`, the doctest fixtures have been split from the main testing suite and doctests are now run using ``$ python -c 'from xclim.testing.utils import run_doctests; run_doctests()'``. (:pull:`1632`).
+* Added ``xclim.indices.generic.season`` to make season start, end and length indices. Added a `stat` argument to ``xclim.indices.run_length.season`` to avoid returning a dataset. (:pull:`1845`).
```

Comment:
https://www.youtube.com/watch?v=P_i1xk07o4g

```suggestion
* Added ``xclim.indices.generic.season`` to make season start, end, and length indices. Added a ``stat`` argument to ``xclim.indices.run_length.season`` to avoid returning a dataset. (:pull:`1845`).
```